app/code/community/Aligent/TimedBanners/Helper/Data.php

```diff
@@ -1,2 +1,58 @@
 <?php
-class Aligent_TimedBanners_Helper_Data extends Mage_Core_Helper_Abstract { }
\ No newline at end of file
+/**
+ * Class Aligent_TimedBanners_Helper_Data
+ */
+class Aligent_TimedBanners_Helper_Data extends Mage_Core_Helper_Abstract {
+    const CONFIG_PREFIX = 'aligent_timedbanners/settings/';
+    const DATETIME_FORMAT = 'd/m/y g:i A';
+
+    /**
+     * @return bool
+     */
+    public function shouldShowBanner()
+    {
+        return Mage::getStoreConfig(self::CONFIG_PREFIX . 'enabled') && $this->isWithinTimedBannerRange();
+    }
+
+    /**
+     * @return string
+     */
+    public function getMessage() {
+        return $this->__(Mage::getStoreConfig(self::CONFIG_PREFIX . 'message', Mage::app()->getStore()));
+    }
+
+    /**
+     * @return DateTime
+     */
+    protected function getStart()
+    {
+        return DateTime::createFromFormat(self::DATETIME_FORMAT, Mage::getStoreConfig(self::CONFIG_PREFIX . 'start'));
```

Comment:
- Good pickup, I've been considering making the input only editable via the Datepicker widget however I haven't seen any resources on achieving this so will require some extra investigation
- I've now switched the locale settings over to use the internal Magento functionality so behavior should now be consistent.
- Ah that's awesome! I've switched to the internal Magento `Mage::app()->getLocale()` functionality now which as above will provide significantly more consistent behaviour.